packages/jira-adapter/src/change_validators/field_contexts/field_contexts.ts

```diff
@@ -37,6 +37,9 @@ const getFieldContexts = async (
       }
       return true
     })
+    .filter(async (ref: ReferenceExpression): Promise<boolean> =>
+      (await elementSource.has(ref.elemID)
+      && !(await ref.getResolvedValue(elementSource) instanceof PlaceholderObjectType)))
```

Comment:
I don't think the `PlaceholderObjectType` is needed and I think we better not rely on that
```suggestion
      await elementSource.has(ref.elemID)
```

Reply:
changed to is instance element on ref.value